frontend/src/components/form/api/ApiWrapperSelectAppend.vue

```diff
@@ -0,0 +1,149 @@
+<template>
+  <div class="ec-api-select-append">
+    <!-- Success icon after saving -->
+    <div class="ec-api-select-append__icon"></div>
+
+    <div class="d-flex gap-1" style="margin-top: -5px">
+      <!-- Retry/Cancel button if saving failed -->
+      <template v-if="wrapper.hasServerError">
+        <v-tooltip bottom class="ml-auto">
+          <template #activator="{ on }">
+            <v-btn
+              fab
+              dark
+              depressed
+              x-small
+              color="error"
+              type="submit"
+              :aria-label="$tc('global.button.tryagain')"
+              v-on="on"
+              @click="wrapper.on.save"
+            >
+              <v-icon>mdi-refresh</v-icon>
+            </v-btn>
+          </template>
+          <span>{{ $tc('global.button.tryagain') }}</span>
+        </v-tooltip>
+        <v-tooltip v-if="resettable" bottom class="ml-auto">
+          <template #activator="{ on }">
+            <v-btn
+              fab
+              dark
+              depressed
+              x-small
+              color="grey"
+              :aria-label="$tc('global.button.cancel')"
+              v-on="on"
+              @click="wrapper.on.reset"
+            >
+              <v-icon>mdi-close</v-icon>
+            </v-btn>
+          </template>
+          <span>{{ $tc('global.button.cancel') }}</span>
+        </v-tooltip>
+      </template>
+
+      <template v-else-if="!wrapper.autoSave">
+        <v-tooltip v-if="wrapper.dirty" bottom class="ml-auto">
+          <template #activator="{ on }">
+            <v-btn
+              fab
+              dark
+              depressed
+              x-small
+              color="success"
+              type="submit"
+              class="mr-1"
+              :aria-label="$tc('global.button.save')"
+              v-on="on"
+            >
+              <v-icon>mdi-check</v-icon>
+            </v-btn>
+          </template>
+          <span>{{ $tc('global.button.save') }}</span>
```

Comment:
Clicking here also opens the dropdown (although, I think that was already the case before)